src/views/tickets/orders.php

```diff
@@ -67,13 +68,18 @@
 	);
 }
 
+$post_type_singular = $post_type ? $post_type->labels->singular_name : _x( 'Post', 'fallback post type singular name', 'event-tickets' );
+
 $is_event_page = class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE === $event->post_type;
 ?>
 
 <div id="tribe-events-content" class="tribe-events-single">
 	<p class="tribe-back">
 		<a href="<?php echo esc_url( get_permalink( $event_id ) ); ?>">
-			<?php printf( '&laquo; ' . esc_html__( 'View %s', 'event-tickets' ), $post_type->labels->singular_name ); ?>
+			<?php
+			// Translators: post type label
+			printf( '&laquo; ' . esc_html__( 'View %s', 'event-tickets' ), $post_type_singular );
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$post_type_singular'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post_type_singular'.

<sub>hash: 4289f0e2cb991d3d955405b6ac1c28e2</sub>